starter-code/src/viking.js

```diff
@@ -1,11 +1,70 @@
-// Soldier
-function Soldier () {}
+//iteration 1 soldier
+function Soldier(health,strength){
+  this.health = health;
+  this.strength = strength;
+}
 
-// Viking
-function Viking () {}
+Soldier.prototype.attack = function(){
+    return this.strength;
+}
+  
+Soldier.prototype.receiveDamage = function(damage){
+    return this.health -= damage;
+}
+  
+var soldier = new Soldier(300,150);
 
-// Saxon
-function Saxon () {}
+soldier.receiveDamage(4);
 
-// War
-function War () {}
+//iteration 2 viking
+function Viking(name,health,strength){
+  Soldier.call(this,health,strength);
+  this.name = name;
+}
+
+Viking.prototype = Object.create(Soldier.prototype);
+
+Viking.prototype.receiveDamage = function (damage){
+  
+  Soldier.prototype.receiveDamage.call(this,damage);
```

Comment:
BAD! You are already inheriting that method from the soldier constructor, having two methods with the same name among inheriting "classes" means that you should use only one for all classes, and not trying to use one inside the other belonging to the other class